packages/wallet-sdk/src/relay/WalletLinkRelay.ts

```diff
@@ -302,8 +302,6 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
       version: this.options.version,
       darkMode: this.options.darkMode,
       session,
-      connected: false,
-      chainId: this.dappDefaultChain,
```

Comment:
not sure how this is being used 🤔

Reply:
these 2 are for the QR code dialog. and chainId is for when dapp provides a preferred default network that is not Ethereum

Reply:
Tested with test-dapp, and RN, RN correctly parsed the chainId and connected via qr code w/o issue.